webapp/app/js/perf.js

```diff
@@ -6,6 +6,19 @@
 
 var perf = angular.module("perf", ['ui.router', 'ui.bootstrap', 'treeherder']);
 
+perf.factory('isReverseTest', [ function() {
```

Comment:
We should really be picking this up in the signature. Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1158233 for this